system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -189,38 +189,38 @@
                         :update-value {:Category "EARTH SCIENCE"
                                        :Term "MARINE SEDIMENTS"
                                        :Topic "OCEANS"}}]
+    ;; Initiate bulk update that shouldn't add anything, including duplicates.
```

Comment:
My only confusion is at this comment here - This bulk update won't add because the update-value already exists right? so it shouldn't add because it's a duplicate...  What do you mean by "including duplicates"?   Are the anything else the prevents the bulkupdate from happening?

Reply:
I meant it shouldn't add duplicates, is there anything else it shouldn't add?

Reply:
the bulk update completes, but no keywords are added.

Reply:
That specific update, before this ticket, would have added a duplicate keyword

Reply:
I understand that :)